src/raster/r.pi/r.pi.neigh/r.pi.neigh.html

```diff
@@ -23,16 +23,12 @@ <h2>SEE ALSO</h2>
 <a href="r.pi.html">r.pi</a>
 </em>
 
-<h2>AUTHORS</h2>
+<h2>AUTHOR</h2>
```

Comment:
```suggestion
<h2>AUTHORS</h2>
```

Reply:
All the r.pi.* should be changed back to AUTHORS as there are more than one listed

Reply:
Yes, will do.